kmir/src/tests/nix/sum-to-n.run.out

```diff
@@ -1,399 +1,398 @@
-<generatedTop>
-  <k>
-    #halt ~> .
-  </k>
-  <returncode>
+<generatedTop>[0m
+  <k>[0m
+    #halt[0m ~> .
+  </k>[0m
+  <returncode>[0m
```

Comment:
I'm not entirely sure what these strange symbols are instead of whitespace, they don't look like anything we want to push to the repo. But they don't appear to change the nix tests success, so they must only be interpreted as whitespace as far as the diff checker is concerned.

Reply:
There suppose to be no changes to the execution results, except that instead of calling `krun`, I did `kparse` first and call the llvm executable with the in memory output. 

I am interested in how these strage symbols coming from, I am suspecting it is due to incomplete support of writing out of data structure memory comparing to writing out a string.

I'd like to create a separate issue to investigate this problem.